.github/workflows/validate.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-20.04", "macos-latest", "windows-latest"]
-        ghc: ["9.4.2", "9.2.3", "9.0.2", "8.10.7", "8.8.4", "8.6.5", "8.4.4"]
+        ghc: ["9.6.1", "9.4.2", "9.2.3", "9.0.2", "8.10.7", "8.8.4", "8.6.5", "8.4.4"]
```

Comment:
It might have changed, but half a year ago 9.4.4 was not provided on older GHA Linux containers and new containers that had 9.4.4 were crashing our CI due some C libraries conflict. https://github.com/haskell/cabal/issues/8630

I do hope this is fixed now, though.

Reply:
Thanks for the explanation @Mikolaj !
As bumping other GHCs to their latest minor version isn't topic of this PR, it could also be done in a separate one, of course...

Reply:
Done, thanks all! I think it makes sense to bump minor versions on other series when you add a new one just because I can't think of a better rule to make sure that the minor versions are ever get bumped!